src/sourceFiles/index.ts

```diff
@@ -697,11 +697,13 @@ export namespace SourceFilesModel {
 
     export interface GeneralExportOptions {
         exportPattern: string;
+        exportQuotes?: ExportQuotes;
     }
 
     export interface PropertyExportOptions {
         escapeQuotes: EscapeQuotes;
         exportPattern: string;
+        exportQuotes?: ExportQuotes;
     }
```

Comment:
These types shouldn't be modified. You need to create a separate one - `JavaScriptExportOptions`

Reply:
I will do that now